app/views/shared/_sidebar.html.erb

```diff
@@ -11,30 +11,29 @@
       <% @categories.each do |category|%>
       <div class="panel panel-default">
         <div class="panel-heading">
-          <h4 class="panel-title"><%= link_to category.name, products_path(get_category_id: category.id)%></h4>
+          <h4 class="panel-title">
+            <%= link_to category.name, products_path(get_category_id: category.id)%>
+          </h4>
         </div>
       </div>
       <% end %>
     </div>
     <!--/category-products-->
 
-    <div class="brands_products">
-      <!--brands_products-->
+    <div class="brands_products"><!--brands_products-->
       <h2><%= t("label.new_item") %></h2>
       <div class="brands-name">
         <ul class="nav nav-pills nav-stacked">
-          <% @categories.each do |category|%>
-          <li>
-            <div class="shipping text-center">
-              <!--shipping-->
-              <%= image_tag("food1.jpg", :alt => "food", :class => "girl img-responsive", size: "80x80") %>
-            </div>
-            <!--/shipping-->
-          </li>
+          <% @new_product.each do |item|%>
```

Comment:
@new_product -> đặt tên biến số nhiều nha, như này đang nhầm tưởng 1 cái new nào đó thôi

Reply:
chưa fix nè (tát)

Reply:
=((( Fix lộn branch chị ơi (Cơ mà lỗi này thì còn to hơn cả lỗi quên fĩx nữa) =)))